Tests/MongoSwiftTests/MongoClientTests.swift

```diff
@@ -26,7 +26,7 @@ final class MongoClientTests: XCTestCase {
         }
 
         let client_t = mongoc_client_new_from_uri(uri)
-        if client_t == nil {
+        guard client_t != nil else {
```

Comment:
I wonder if we can combine this with the assignment above. I believe `guard let client_t = mongoc_client_new_from_uri(uri)` should only fail if `nil` is returned.

Reply:
We can, and we did :).